src/DummyConsoleApp/RunProcessAsTask.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>DummyConsoleApp</name>
+    </assembly>
+    <members>
+    </members>
+</doc>

```

Comment:
This may have just been done without me adding it to git ignore. I was follwing the .net core guide to NuGet packaging. This is uneeded.